api/envoy/extensions/outlier_detection_monitors/common/v3/error_types.proto

```diff
@@ -40,5 +45,6 @@ message ErrorBuckets {
   repeated LocalOriginErrors local_origin_errors = 2;
 
   // List of buckets "catching" database errors.
+  // [#not-implemented-hide:]
   repeated DatabaseErrors database_errors = 3;
```

Comment:
It was originally added to illustrate how new error types can be added. I think it does no harm to keep it to be honest. That code can only be sent by terminal database filter (like redis) and it currently does not send that code, so it is no-op. I am planning to add support for databases as next step and will have to bring it back.

Reply:
I am more in favor of adding fields into api together with the relevant PRs/implementation, otherwise there is a risk of having too many unimplemented features in api since people tend to be busy with work/can change priorities. cc @wbpcode